app/_data/kong_versions.yml

```
@@ -429,7 +429,12 @@
   endOfLifeDate: "2026-04-19"
   branch: release-2.7
   lts: true
+- edition: mesh
+  version: 2.8.0
+  release: 2.8.x
+  branch: release-2.8
```

```suggestion
  branch: release-2.8
  latest: true
```
looks like the `latest` label got lost somewhere along the way, so 2.7 is still displaying as the default latest version: https://deploy-preview-7557--kongdocs.netlify.app/mesh/

---

It wasn't released at this time. Now it should be ok